### DISCUSSION TITLE
RC Transmitter using RadioLib ( SX1280 )

### DISCUSSION THREAD

**Discussion by alx-uta** [jgromes/RadioLib#922] [Show and tell]
After reading different tutorials about RC Transmitters using Arduino, I decided to make my own.
The main library used in my project is RadioLib.

Thank you for creating and sharing this library.

![rc_transmitter_all](https://github.com/jgromes/RadioLib/assets/16883821/ec63a6fb-2ea5-45f9-9522-eaadf6b0c1e6)

GitHub: https://github.com/alx-uta/RC-Transmitter

**Comment by jgromes** [comment]
@alx-uta that looks really cool, thanks for sharing! And I'm glad you found this library useful :)

**Reply by alx-uta** [reply]
Thanks!
